<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="10" hostname="ip-172-31-22-110" name="com.simplilearn.AmazonLoginTests" tests="11" failures="0" timestamp="2021-10-24T07:13:00 UTC" time="0.007" errors="1">
  <testcase name="adminLogoutTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- adminLogoutTest -->
  <system-out/>
  <testcase name="adminLoginTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- adminLoginTest -->
  <system-out/>
  <testcase name="childTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- childTest -->
  <system-out/>
  <testcase name="beforeSuite" time="0.007" classname="com.simplilearn.AmazonLoginTests">
    <error type="java.lang.RuntimeException" message="some unknown err....">
      <![CDATA[java.lang.RuntimeException: some unknown err....
at com.simplilearn.AmazonLoginTests.beforeSuite(AmazonLoginTests.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- beforeSuite -->
  <system-out/>
  <testcase name="adminDashboardTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- adminDashboardTest -->
  <system-out/>
  <testcase name="userLogoutTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- userLogoutTest -->
  <system-out/>
  <testcase name="longRunningTesCase" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- longRunningTesCase -->
  <system-out/>
  <testcase name="userLoginTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- userLoginTest -->
  <system-out/>
  <testcase name="parentTest" time="0.000" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- parentTest -->
  <system-out/>
  <testcase name="axminLoginTest" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- axminLoginTest -->
  <system-out/>
  <testcase name="aaminLoginTest" classname="com.simplilearn.AmazonLoginTests">
    <skipped/>
  </testcase> <!-- aaminLoginTest -->
  <system-out/>
</testsuite> <!-- com.simplilearn.AmazonLoginTests -->
